PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/task/CreateStorageAccountTask.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.storage.task;
+
+import com.microsoft.azure.toolkit.intellij.common.Draft;
+import com.microsoft.azure.toolkit.lib.Azure;
+import com.microsoft.azure.toolkit.lib.common.model.ResourceGroup;
+import com.microsoft.azure.toolkit.lib.resource.AzureGroup;
+import com.microsoft.azure.toolkit.lib.storage.model.StorageAccountConfig;
+import com.microsoft.azure.toolkit.lib.storage.service.AzureStorageAccount;
+import com.microsoft.azure.toolkit.lib.storage.service.StorageAccount;
+import com.microsoft.azuretools.ActionConstants;
+import com.microsoft.azuretools.telemetry.TelemetryConstants;
+import com.microsoft.azuretools.telemetrywrapper.ErrorType;
+import com.microsoft.azuretools.telemetrywrapper.EventType;
+import com.microsoft.azuretools.telemetrywrapper.EventUtil;
+import com.microsoft.azuretools.telemetrywrapper.Operation;
+import com.microsoft.azuretools.telemetrywrapper.TelemetryManager;
+
+import java.util.Collections;
+
+public class CreateStorageAccountTask {
+
+    private final StorageAccountConfig config;
+
+    public CreateStorageAccountTask(StorageAccountConfig config) {
+        this.config = config;
+    }
+
+    public StorageAccount execute() {
+        final Operation operation = TelemetryManager.createOperation(ActionConstants.StorageAccount.CREATE);
+        try {
+            operation.start();
+            final String subscriptionId = config.getSubscription().getId();
+            EventUtil.logEvent(EventType.info, operation, Collections.singletonMap(TelemetryConstants.SUBSCRIPTIONID, subscriptionId));
```

Comment:
avoid using ventUtil.logEvent

Reply:
All resources used this code. let us consider to refactor in a seperate task.